content/source/docs/cloud/integrations/service-now/developer-reference.html.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: "cloud"
+page_title: "Developer Reference - ServiceNow Service Catalog Integration -
+Terraform Cloud and Terraform Enterprise"
+description: |-
+  Developer reference for the ServiceNow Service Catalog Integration.
+---
+
+## Terraform Variables and ServiceNow Variable Sets
```

Comment:
This page needs an H1 per-page header above the first H2 per-section header; otherwise it'll,

![image](https://user-images.githubusercontent.com/484309/91224494-0dacb180-e6d7-11ea-8849-c0082e63f1ad.png)

It also probably needs a sentence saying what this page is for.